js/src/lib/components/InferenceWidget/shared/helpers.ts

```diff
@@ -89,6 +90,7 @@ async function callApi(
 			method: "POST",
 			body,
 			headers,
+			credentials: includeCredentials ? "include" : null,
```

Comment:
```suggestion
			credentials: includeCredentials ? "include" : "same-origin",
```
^^ let's use the default value rather